runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsConfig.java

```diff
@@ -302,7 +302,6 @@ default Color insaneValueColor()
 	{
 		return Color.decode("#FF66B2");
 	}
-
```

Comment:
Please stop changing formatting in unrelated parts of files in your prs.

Reply:
Sorry, that one must of snuck through when rebasing.